src/searchForm/autoComplete/overlay/index.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components'
+import { color, radius, space } from '_utils/branding'
+
+import autoComplete from './AutoCompleteOverlay'
+
+const overlayWidth = '375px'
+
+const StyledAutoComplete = styled(autoComplete)`
+  & {
+    box-sizing: border-box;
+    width: ${overlayWidth};
+    border: 1px solid ${color.border};
+    border-radius: ${radius.l};
+    padding: ${space.xl};
+    /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
```

Comment:
👍 good point. Is it something planned for a next PR?

Reply:
This class and its comment are copied from @wakooka's original overlay component
We've planned to factorize some code in future PR and this update should be added in this next PR